docs/superpowers/specs/2026-08-06-web-ui-structure-design.md

```
@@ -0,0 +1,214 @@
+
+---
+
+## 4. 搬家表（from → to，已有码）
+
+| 今日 | 目标 |
+|---|---|
+| `lib/chat/*` | `features/chat/*`（或 `features/chat/lib/*` 若需子层） |
+| `lib/route-detail/*` | `features/route-detail/*` |
+| `components/route-detail/*` | `features/route-detail/ui/*` |
+| `components/auth/*` | `features/auth/ui/*` |
+| `components/landing|home|Splash|Landing` | `features/landing/*` |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 2; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)